mongodb/nodejs/linux/README.md

```diff
@@ -0,0 +1,37 @@
+# Node.js express using CosmosDB Emulator Linux Docker
+
+This sample app is a notes app that uses the MongoDB Node.js Driver to connect to a CosmosDB MongoDB API instance running in a Linux Docker container. The sample creates basic CRUD operations.
```

Comment:
_This sample application is a note-taking app that leverages the MongoDB Node.js Driver to establish a connection with a CosmosDB MongoDB API instance, which is hosted within a Linux Docker container. This sample showcases fundamental CRUD (Create, Read, Update, Delete) operations._